ecosystem_tests/dorkl.py

```diff
@@ -242,6 +249,12 @@ def prepare_test(plugins=None, secrets=None, plugin_test=True,
     if pip_packages:
         pip_command = pip_command + ' '.join(pip_packages)
     docker_exec(pip_command)
+    if use_vpn:
+        value = base64.b64decode(os.environ['vpn_config'])
```

Comment:
do we have plans to do py3 compatibility for this code?
if we do, we will have to encode 'vpn_config' then decode again